libra-node/src/main_node.rs

```diff
@@ -19,29 +33,17 @@ use libra_json_rpc::bootstrap_from_config as bootstrap_rpc;
 use libra_logger::prelude::*;
 use libra_mempool::MEMPOOL_SUBSCRIBED_CONFIGS;
 use libra_metrics::metric_server;
-use libra_types::{on_chain_config::ON_CHAIN_CONFIG_REGISTRY, waypoint::Waypoint, PeerId};
+use libra_types::{on_chain_config::ON_CHAIN_CONFIG_REGISTRY, PeerId, waypoint::Waypoint};
 use libra_vm::LibraVM;
 use libradb::LibraDB;
 use network::validator_network::network_builder::{
-    AuthenticationMode, NetworkBuilder, HANDSHAKE_VERSION,
+    AuthenticationMode, HANDSHAKE_VERSION, NetworkBuilder,
```

Comment:
in general I would expect cargo x fmt to call this out, was this done manually?

Reply:
see "optimize includes" previously.  I believe I ran cargo x fmt, but will redo to confirm.